src/main/java/com/academy/ui/pages/ClubCardComponent.java

```diff
@@ -28,7 +28,7 @@ public class ClubCardComponent extends BaseComponent {
     @FindBy(xpath = ".//a[contains(@class,'details-button')]")
     protected WebElement detailsButton;
 
-    @FindBy(xpath = "./descendant::span[@class='ant-tag tag css-k8nnl9'][1]")
+    @FindBy(xpath = "./descendant::div[@class='tags club-tags']")
```

Comment:
List<DirectionTagComponent> directions are the list of elements for categories.  (The naming is problematic: we agreed to used word "direction" in the begging, but now it is a bit misleading)
But the single element for categories cannot be used in any case: there can be more then one category in the card. So it Always should be a list.

Reply:
I got your point, so I delete method and left xpath List<WebElement> categoryClub. 
I suggest that it could be better to left your xpath List<WebElement> directionTags and delete anotation AccessLevel.NONE, how about it?

Reply:
There is already List<DirectionTagComponent>directions. And DirectionTagComponent is a component for categories with name and icon. 
But if you need, you can delete the AccessLevel.NONE